pkg/api/api.go

```diff
@@ -70,6 +73,17 @@ func (a *API) ProfileWithVersionHandler(w http.ResponseWriter, r *http.Request)
 	marshalResponse(w, result)
 }
 
+// ProfileGreaterThanVersionHandler is the handler for /profiles/{catalog}/{profile}/{version} requests.
+func (a *API) ProfileGreaterThanVersionHandler(w http.ResponseWriter, r *http.Request) {
+	sourceName, profileName, catalogVersion := mux.Vars(r)["catalog"], mux.Vars(r)["profile"], mux.Vars(r)["version"]
+	result := a.profileCatalog.GetGreaterThan(sourceName, profileName, catalogVersion)
+	if result == nil {
+		w.WriteHeader(404)
+		return
+	}
+	marshalResponse(w, result)
+}
+
```

Comment:
do we need to check for empty strings? This might apply to other funcs in here too

Reply:
Sure, why not. :)

Reply:
Let's skip that because that means changes in PCTL to properly show a 400. Because 404 is not very valid in case parameters are incorrect.
Created https://github.com/weaveworks/profiles/issues/140 to track

Reply:
:+1: